model/item.rb

```diff
@@ -1,13 +1,9 @@
 class Item
-  attr_reader :id, :archived
+  attr_reader :genere, :author, :source, :label
```

Comment:
well done in this class, however, in the diagram class, it's named "genre" instead "genere", please change the name.

Reply:
Thanks for mentioning it.